incubator/jaeger/templates/query-deploy.yaml

```diff
@@ -100,6 +115,20 @@ spec:
             subPath: {{ .subPath }}
             readOnly: {{ .readOnly }}
         {{- end }}
+        {{- if .Values.storage.cassandra.tls.enabled }}
+          - name: {{ .Values.storage.cassandra.tls.secretName }}
+            mountPath: "/cassandra-tls/ca_cert.pem"
+            subPath: "ca_cert.pem"
+            readOnly: true
+          - name: {{ .Values.storage.cassandra.tls.secretName }}
+            mountPath: "/cassandra-tls/client_cert.pem"
+            subPath: "client_cert.pem"
+            readOnly: true
+          - name: {{ .Values.storage.cassandra.tls.secretName }}
+            mountPath: "/cassandra-tls/client_key.pem"
+            subPath: "client_key.pem"
```

Comment:
I think we might want to use kebab-case instead of snake_case  for consistency.

Reply:
Right.